app/services/export_offenders.rb

```diff
@@ -0,0 +1,12 @@
+module ExportOffenders
+  module_function
+
+  def call(file)
+    require 'csv'
+
+    CSV.open(file, 'wb') do |csv|
+      csv << %w(noms_offender_id identity_id offender_id)
+      Identity.all.each { |identity| csv << [identity.noms_offender_id, identity.id, identity.offender.id] }
```

Comment:
I would also double check what is default order of the output, **as if none is enforced**, you might [end up find two export of the same data producing different generated files](http://stackoverflow.com/questions/4986496/activerecord-find-all-not-sorting-by-id).

It is probably recommended you enforce an order for retrieving the records.